templates/emails/ng_alert_notification.txt

```diff
@@ -0,0 +1,28 @@
+{{Subject .Subject "{{.Title}}"}}
```

Comment:
These are copied as is, for now, we'll have to re-visit all templates as we unify so don't read too much into it.